app/assets/stylesheets/navbar.scss

```diff
@@ -28,4 +28,22 @@
   .navbar-collapse {
     border-top: 0;
   }
+
+  @media screen and (min-width: 768px) and (max-width: 991px){
```

Comment:
One thing I would change, though, is remove `screen` here. It's not being specified in any of the other media queries on the site, and it's not necessary for this.